index.js

```diff
@@ -1,3 +1,4 @@
 import menu from "./menu.js";
 
 // TODO
+menu.addDishToCourse('mains', 'Palov', 35);
```

Comment:
It is not finished yet.

Reply:
So sorry, it is all because of leaderboard rush)))))

Reply:
Quality should be #1 priority.